libs/portals/shared-modules/delegations/src/lib/messages.ts

```diff
@@ -6,10 +6,14 @@ export const m = defineMessages({
     defaultMessage: 'Veldu umboð',
     description: 'Choose delegation',
   },
-  noDelegations: {
-    id: 'sp.access-control-delegations:empty',
+  noOutgoingDelegations: {
+    id: 'sp.access-control-delegations-outgoing:empty',
```

Comment:
We traditionally try to match the namespaces of the id for each translation file (with some unique exceptions) so the `-outgoing` and `-incoming` should be after the `:` (`'sp.access-control-delegations:empty-outgoing'`)